tests/e2e/utils/helpers.go

```diff
@@ -1071,3 +1073,12 @@ func GetKeyTransferFee(output string) (uint64, error) {
 func GetAPILargeContext() (context.Context, context.CancelFunc) {
 	return context.WithTimeout(context.Background(), constants.APIRequestLargeTimeout)
 }
+
+func GetE2EHostInstanceID() (string, error) {
+	hosts, err := ansible.GetInventoryFromAnsibleInventoryFile(path.Join(GetBaseDir(), constants.NodesDir, constants.AnsibleInventoryDir, constants.E2EClusterName))
+	if err != nil {
+		return "", err
+	}
+	_, cloudHostID, _ := models.HostAnsibleIDToCloudID(hosts[0].NodeID)
```

Comment:
as we are only getting IP address/ SSH key pairs from the user we should not use CloudID and use IP addresses instead